packages/ncids-js/src/components/nci-header/utils/mega-menu/mega-menu-nav.ts

```diff
@@ -440,24 +440,24 @@ export class MegaMenuNav {
 
 		if (results) {
 			clearTimeout(timer);
-		}
 
-		// Programmatically create unique id
-		const id = `menu-${path.toString().replace(/[^\w\s]/gi, '')}`;
-		this.content = results;
-		this.content.setAttribute('id', id);
-		this.content.classList.add('hidden');
-		this.content.ariaLive = 'polite';
-		this.content.ariaBusy = 'false';
-		this.content.ariaAtomic = 'true';
-
-		// Replace loading container with menu
-		this.loader.classList.add('hidden');
-		this.loaderContainer.classList.add('hidden');
-		this.loaderContainer.replaceWith(this.content);
-
-		// Set button aria-controls to new content
-		button.setAttribute('aria-controls', id);
+			// Programmatically create unique id
```

Comment:
There seems to be a need for a update of the unit tests for mega-menu? I assume this is being done in the if to not draw the item if it is null. Although, I don't know where it is drawing either.